spec/functional_spec.rb

```diff
@@ -29,6 +29,13 @@
     end
   end
 
+  after :all do
+    KeyringLiberator.get_all_keyrings_named('TestProject').each do |keyring|
+      file = KeyringLiberator.yaml_path_for_path(keyring.path)
+      FileUtils.rm(file) if File.exist?(file)
+    end
+  end
```

Comment:
I see, I think this is a fair trade-off, we are touching the file system in tests